.ci/openshift-ci/buildall_install.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Build kata-containers and install into a given destination directory
+#
+
+set -e
+
+if [ -z "$1" ];then
+	echo "Usage: $0 path/to/install/dir"
+	exit 1
+fi
+
+export DESTDIR=$1
+echo "Build and install kata-containers at: ${DESTDIR}"
+
+export PATH=${GOPATH}/bin:$PATH
+
+cidir=$(dirname "$0")/..
+source /etc/os-release || source /usr/lib/os-release
+source "${cidir}/lib.sh"
+
+# The scripts rely on sudo which is not installed in the build environment.
+yum install -y sudo
+"${cidir}/setup_env_centos.sh" default
```

Comment:
Similar - what if this is run on RHEL or Fedora? Please can you add a check and die if the correct distro isn't found.

Reply:
ok, I will add this check.

Reply:
I added this check and a comment as suggested by @c3d